### ISSUE TITLE
[MU4 Issue] I found a score that makes MS4 crash but that works fine in MS3.6.2

### ISSUE THREAD

**Issue by Stccl** [musescore/MuseScore#13810]
**Describe the bug**
This score (https://mega.nz/file/H48GkQZK#16DHPXmUA8NShfjXipswPFDhMeITV1MNmY3B8PlmcG8) makes MS4 crash (program doesn't respond) when I try to open it, but i can open it without any issue with the latest release of MS3.

**To Reproduce**
1. Download the score from the link I provided earlier or here: https://musescore.com/user/38235231/scores/5275054 (not mine)
2. Try to open it with MS4, and then MS3 to compare.

**Expected behavior**
I assume it should be able to open it and ask if I want to use the new font and stuff.

**Platform information**
 - Issue occured on both of my 2 PCs which use respectively Windows 10 & 11.

**Additional context**
I haven't tested on Linux or MacOS so I suggest doing that.

**Comment by krtschil** [comment]
Checked on Linux: same as described above. After selecting the score the app hangs and after a few minutes the system asks either to wait longer or to kill the app. Waiting did not change anything.
The score was created with MU 2.3.2. I opened it in 3.6.2 and saved it in that format. Opening that version in MU4 was no problem.

It seems that MU4 has problems opening scores from v2.3.2 (if it is supported at all).
